src/Components/Details/ShowDetails.js

```diff
@@ -1,14 +1,15 @@
 import React, { Component } from 'react';
 import { fetchSingleMovie } from '../../APICalls';
+import { NavLink } from 'react-router-dom';
 import './ShowDetails.css';
 
 class ShowDetails extends Component {
-  constructor({ movie }, props) {
-    super();
+  constructor(props) {
```

Comment:
Lets make a refactor branch and look at the number of properties in here again together.